eng/pipelines/templates/steps/analyze.yml

```diff
@@ -10,6 +10,7 @@ steps:
 
   - task: Powershell@2
     displayName: 'Dependency Check'
+    condition: false
```

Comment:
```suggestion
    continueOnError: true
```

You could also just continue on error so the step runs but doesn't fail the build.

Reply:
Good idea, fixed.